# Enhance test runner so that needed perl interpreter may be selected
# according to PATH environment variable.
--- a/test/run	2022-07-23 18:18:44.842195625 +0200
+++ b/test/run	2022-07-23 18:35:51.455375370 +0200
@@ -1,4 +1,6 @@
-#!/usr/bin/perl -w -U
+#!/usr/bin/env perl
+
+use warnings;
 
 # Copyright (c) 2007, 2008 Andreas Gruenbacher.
 # All rights reserved.
